Gemfile

```diff
@@ -13,7 +13,7 @@ group :development do
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'rake', '10.4.2'
-  gem 'rubocop'
+  gem 'rubocop', '~> 0.43.0'
```

Comment:
ARGH, one of course has to delete the _Gemfile.lock_ as well. Otherwise even with deleting the gems, you only get the old versions back. Make sense now.